src/main/java/com/taskadapter/redmineapi/internal/RedmineJSONBuilder.java

```diff
@@ -75,6 +75,7 @@ static void writeTimeEntry(JSONWriter writer, TimeEntry timeEntry)
 		addIfSetShort2(writer, "spent_on", storage, TimeEntry.SPENT_ON);
 		addIfSetFullDate(writer, "created_on", storage, TimeEntry.SPENT_ON);
 		addIfSetFullDate(writer, "updated_on", storage, TimeEntry.SPENT_ON);
+		addIfSet(writer, "custom_fields", storage, TimeEntry.CUSTOM_FIELDS);
```

Comment:
this gives Redmine Server error 500 in my simple test.
I changed this to 
`writeCustomFields(writer, timeEntry.getCustomFields());`
- again, similarly to how it is done for other elements, like Project.